docs/users-guide/05-visualizing-results.md

```diff
@@ -227,6 +253,16 @@ There are three main ways to configure axes:
 
 Here's where you can choose to hide the **label** for your x- or y-axis. You can also customize the text for your axes labels here.
 
+### Waterfall charts
+
+Waterfall charts are a kind of bar chart useful for visualizing results that contain both positive and negative values. Each bar on a waterfall chart shows either an increase or decrease, with a final bar on the right of the chart that represents the total value.
+
+![Waterfall chart](images/visualizations/waterfall-chart.png)
+
+In the example above, the waterfall chart displays "Profit" for each "Product": Apples, Bananas, Oranges, Peaches, and Mangos. From left to right, each bar indicates the change in total. The products with green bars indicate positive values (they made a profit). Peaches, however, lost money, indicated by a red bar, which signals a negative value. The purple bar at the end shows the total profit of all products combined. You can show values on each bar, and change the colors for increases and decreases.
```

Comment:
We've changed the color of the Total bar, so we can remove the reference to its color.

```suggestion
In the example above, the waterfall chart displays "Profit" for each "Product:" apples, bananas, oranges, peaches, and mangos. From left to right, each bar indicates the change in total. The products with green bars indicate positive values (they made a profit). Peaches, however, lost money, indicated by a red bar, which signals a negative value. The bar at the end shows the total profit of all products combined. You can show values on each bar, and change the colors for increases and decreases.
```